VVResonances/interactive/makeInputs.py

```diff
@@ -307,14 +313,14 @@
             f.makeSignalShapesMVV("JJ_j2"+str(signal_inuse)+"_"+str(period),signaltemplate_inuse,fixParsSigMVV[signal_inuse],"jj_l2_softDrop_mass <= 105 && jj_l2_softDrop_mass > 85 && jj_l1_softDrop_mass <= 85 && jj_l1_softDrop_mass >= 65")
             #f.makeSignalShapesMVV("JJ_j1"+str(signal_inuse)+"_"+str(period),signaltemplate_inuse,fixParsSigMVV[signal_inuse])
             #f.makeSignalShapesMVV("JJ_j2"+str(signal_inuse)+"_"+str(period),signaltemplate_inuse,fixParsSigMVV[signal_inuse])
-        elif signal_inuse.find("H")==1 or signal_inuse.find("WZ")==1:
-            f.makeSignalShapesMVV("JJ_"+str(signal_inuse)+"_"+str(period),signaltemplate_inuse,fixParsSigMVV[signal_inuse])
+        else:
```

Comment:
I realise that this is probably not the best fix but this way we will create a useless file when running on ZH (WH)  samples, because this else is only wrt to `if signal_inuse.find("WZ")!=-1:` and not on `if signal_inuse.find("H")!=-1:`. 
It took me some time to understand why I was getting extra outputs for the ZprimeZH. Maybe we can leave the `else` here and change the `if` about the WZ to a `elif` ?

Reply:
How about merging the PR and after that you add your fix?

Reply:
yes will do!